manage.py

```
@@ -3,15 +3,93 @@
+            'TOKEN_VALIDATOR_PORT': env.get('TOKEN_VALIDATOR_PORT', '3001'),
+        })
+        
+        # Start the token validator using bash (since it's a shell script)
+        token_validator_process = subprocess.Popen(
+            ['bash', str(token_validator_script)],
+            cwd=str(project_root),
+            env=env,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            preexec_fn=os.setsid  # Create new process group for clean shutdown
+        )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)